lib/reduceBy/index.js

```diff
@@ -0,0 +1,10 @@
+import curryN from '../curryN'
+
+export default curryN(4, function reduceBy(reducer, init, by, list) {
```

Comment:
maybe we have a reason to add `_curry4` if it's lighter

Reply:
I'm not sure the `_curry4` will be lighter (if we'll do it like `_curry2 / curry3`)